kcli_plan.yml

```diff
@@ -23,10 +21,9 @@
   - 01_create_iso.sh
   - 02_install_baremetal.sh
   - sno_params.yml
-  - origin: helpers/racadm.sh
```

Comment:
file is located in helpers/racadm.sh
if you wanted to move it, it is missing from this commit.